### ISSUE TITLE
Headers not importing after first block in import_blocks.py

### ISSUE THREAD

**Issue by hal0x2328** [CityOfZion/neo-python#534]
### Current behavior

- What is the problem?

import_blocks.py *appears* to import blocks successfully, however, the block headers are not stored properly in the database - so when a contract later tries to call `GetHeader`, it will fault for every block, leading to inconsistency in storage as well.

This happens because of the fact that import_blocks.py uses a single instance of the`Block` object and just deserializes the block data into it over and over while loading chain.acc

During this process, in `AddBlockDirectly`, the line `self.AddHeader(block.Header)` is supposed to import the header from the newly deserialized block. But, `Block.Header` uses the following logic:
```
        if not self.__header:
            self.__header = Header(self.PrevHash, self.MerkleRoot, self.Timestamp,
self.Index, self.ConsensusData, self.NextConsensus, self.Script)
```
Since the block has already been initialized with a header after the first block, subsequent block headers will not be added to the header list since `self.__header` already exists.

### Expected behavior

- What should be happening?

import_blocks.py should import headers properly during `AddBlockDirectly` in LevelDBBlockchain.py

I'm not sure of the best solution to solve this issue. I've tried:

Method 1: Use a new block object for each imported block in chain.acc. This works, but slows the import down significantly.

Method 2: Remove the check for an existing `self.__header` in `Block.Header`. This works and is much faster than Method 1, but may impact overall performance of neo-python in general.

It would be nice if we could simply set `block.__header` to `None` before calling Block.Header but it's supposed to be private. 

I'm inclined to add a `useCached=True` parameter to `Block.Header` and set it to false when calling from `AddBlockDirectly`, so I'm seeking input on this option in this issue. I don't think it should impact any other code but I want to know if there is a better way to accomplish this, perhaps making `__header` a public variable?

### How to reproduce

Execute import-blocks.py with a known-good chain.acc file

### Your environment

- OS: OSX High Sierra
- neo-python version: neo-python v0.7.6
- Python version: python 3.6.3

**Comment by ixje** [comment]
Thanks for the report. I agree with your assessment that creating a new block slows it down too much. I believe removing the check will only impact performance when syncing up, because that's the only place where it gets called a lot (the impact might be high though). 

As for `__header` being private, I don't really like the double underscore private declaration. I think we could make it `_header` such that we know it's private, but for the sake of issues like these you can still manipulate it.

**Comment by ixje** [comment]
@hal0x2328 want to PR the fix?

**Comment by localhuman** [comment]
Fixed in #537
